gravitee-am-service/src/main/java/io/gravitee/am/service/impl/I18nDictionaryService.java

```diff
@@ -142,17 +145,22 @@ public Single<I18nDictionary> update(ReferenceType referenceType, String referen
                         .defaultIfEmpty(Optional.empty())
                         .map(optionalDict -> {
                             if (optionalDict.isPresent() && !optionalDict.get().getId().equals(id)) {
-                                throw new DictionaryAlreadyExistsException(updateDictionary.getName(), updateDictionary.getLocale());
+                                throw new DictionaryAlreadyExistsException(updateDictionary.getLocale());
                             }
                             return existingDictionary;
                         })
                 )
                 .flatMapSingle(oldDictionary -> {
                     var toUpdate = new I18nDictionary(oldDictionary);
                     if (updateDictionary.getName() != null) {
+                        validateName(updateDictionary.getName());
```

Comment:
> Same as above

With a PUT the name and/or locale may be null. I think the only way to do this with annotations is for a custom `ConstraintValidator`. WDYT?

Reply:
In that case, if it's normal to have a null value for name and/or locale we can't use annotation easily. Bu, instead of testing null in the if and after test the name with validateName we can use validateName directly in the if, no ?